### DISCUSSION TITLE
New Report Reason for broken Measured Bar

### DISCUSSION THREAD

**Discussion by SporyTike** [RetroAchievements/RAWeb#788] [Ideas]
Would it be possible to add a new report reason for tickets? Now Measured is a thing and it allows the player to see the current progress of the achievement if the developer added measured to the logic. I already found some achievements with broken bars and I think that could mean the achievement itself is broken but I can't confirm because they still need a few hours of playing to actually unlock. I don't want to open tickets with "wrong trigger" or "doesn't trigger" just because maybe the visual progressbar is broken so it would be nice to have - at least on the site - the possibility to create tickets with a reason like "Broken Progressbar".

**Comment by lonegraywolf2000** [comment]
I second this issue. I created [this ticket](https://retroachievements.org/ticketmanager.php?i=53645) for [The Legend of Zelda](https://retroachievements.org/game/1454) when I saw the problem. No need for me to re-create this issue when it already exists.

**Comment by redwizard42** [comment]
This would definitely be helpful.  "Trigger icon showing at wrong time(s)" would be another good similar addition.
